From 8b0c47d3e7604a4651d654e435e001274d07177c Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Thu, 20 Jan 2022 21:27:48 +0100
Subject: [PATCH] Fix openssh-7.2p2-x11.patch for macOS

---
 gsi_openssh/source/channels.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gsi_openssh/source/channels.c b/gsi_openssh/source/channels.c
index 8c68cb7..27bf695 100644
--- a/gsi_openssh/source/channels.c
+++ b/gsi_openssh/source/channels.c
@@ -4811,7 +4811,7 @@ x11_connect_display(struct ssh *ssh)
 			debug("x11_connect_display: $DISPLAY is launchd");
 
 			/* Create a socket. */
-			sock = connect_local_xsocket_path(path);
+			sock = connect_local_xsocket_path(path, sizeof(path));
 			if (sock < 0)
 				return -1;
 
-- 
2.29.2

